cli/crates/typed-resolvers/src/lib.rs

```diff
@@ -56,3 +61,20 @@ pub fn check_resolvers(resolvers_root: &Path, schema: &analyze::AnalyzedSchema<'
 
     AnalyzedResolvers { errs }
 }
+
+fn experimental_codegen_is_true(parsed_schema: &engine_parser::types::ServiceDocument) -> bool {
```

Comment:
is_true feels a bit weird on a boolean function, "enabled" perhaps?

Reply:
Good point, doing this

Reply:
Done and pushed